index.html

```diff
@@ -268,14 +271,67 @@ <h3>
           <li><a href="">Web development</a></li>
           <li><a href="">API design</a></li>
         </ul>
-    
+
       </div>
 
     </div>
 
 
   </section>
-  
-    
+
+  <section class="contact-form">
+    <div class="contact-form-text">
+      <h2 class="contact-heading">
+        I'm always interested in hearing about new projects,
+        so if you'd like to chat please get in touch.
+      </h2>
+    </div>
+
+    <form action="https://formspree.io/f/mqkoykyv" method="POST" class="contact-form-container">
+
+      <label>
+        <input type="text" name="full-name" placeholder="Full name">
+      </label>
+        
+      <label>
+        <input type="email" name="email" placeholder="Email address">
+      </label>
+        
+        
+      <textarea name="" id="" rows="10" placeholder="Hey, I would like to connect to..."></textarea>
+        
+        
+      <button type="submit"> Get in touch </button>
+
+    </form>
```

Comment:
- [ ] Please I highly recommend implementing HTML validations to the form field per project requirements. 😃 
   - Name input (can be first name and last name field depending 
      on the template used) has a `maxlength="30"` attribute.
    - Message input has `maxlength="500"` attribute.
    - All input fields have the `required` attribute to activate the 
       validation on submission.
    - Please this is an added screenshot of the specific project 
      requirements. 😃 

![Screenshot (119)](https://user-images.githubusercontent.com/78104544/228726499-3ccf44f8-0af8-4627-a714-22529305ddc4.png)